docs/helm/configure-namespaced-helm-repos.md

```diff
@@ -1,85 +1,55 @@
-# Configure Namespace Scoped Chart Repositories using namespaced CRDs
+# Configure Namespace-scoped Helm Chart Repositories Using Namespaced CRDs
 
-While the cluster-scoped CRD `HelmChartRepository` for Helm repository provides ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
+While the cluster-scoped [HelmChartRepository](https://github.com/openshift/api/blob/master/helm/v1beta1/0000_10-helm-chart-repository.crd.yaml) CRD for Helm repository provides the ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
 
-To avoid permission elevation, use namespace scoped CRD `ProjectHelmChartRepository`.
+The namespace-scoped `ProjectHelmChartRepository` custom resource allows project members with the appropriate RBAC permissions to create Helm repository resources of their choice but scoped to their namespace. Such project members can see charts from both clusters-scoped and namespace-scoped Helm repositories.
 
-## Basic structure of `ProjectHelmChartRepository`
+_Note_: Administrators can limit users from creating namespace-scoped Helm repository resources to prevent access to unauthorized services or applications.
+
+## Basic structure of `ProjectHelmChartRepository` CR
 
 ```yaml
 apiVersion: helm.openshift.io/v1beta1
 kind: ProjectHelmChartRepository
 metadata:
-  name: my-enterprise-chart-repo
+  name: NAME
 spec:
   url: https://my.chart-repo.org/stable
 
   # optional name that might be used by console
-  name: myChartRepo
+  name: CHART_REPO_DISPLAY_NAME
 
   # optional and only needed for UI purposes
-  description: my private chart repo
+  description: My private chart repo
 
   # required: chart repository URL
   connectionConfig:
     url: HELM_CHART_REPO_URL
 ```
 
-## Getting Started with `ProjectHelmChartRepository`
+## Adding Namespace-scoped Custom Helm Chart Repositories
 
-To add a new namespace-scoped Helm repository `stable` to a desired namespace `my-namespace`:
+To add a new namespace-scoped Helm repository, add a custom Helm Chart Repository CR, for example, `azure-sample-repo` to your `my-namespace` namespace:
 
 ```bash
 $ cat <<EOF | oc apply --namespace my-namespace -f -
```

Comment:
Suggestion: I believe that in upstream we use only `kubectl` and not `oc`. `oc` is only for downstream. (Global change for all such `oc` occurrences)

Reply:
Updated :+1: